packages/unplugin/src/plugin/core.ts

```diff
@@ -125,27 +129,66 @@ export const unpluginFactory: UnpluginFactory<PandaPluginOptions | undefined> =
   }
 
   let server: ViteDevServer
-  let lastCss: string | undefined
+
+  let latestCss: string | undefined
+  // Necessary because when using virtual:pandacss in cases with many files,
+  // the css may be empty the first time the page is loaded, and HMR doesn't work if the page hasn't loaded yet.
```

Comment:
I need to do more testing here. This might only be necessary the first time css is loaded, or it might not be necessary at all if we keep track of if the worker has anything in progress and await it.